src/dbc.rs

```diff
@@ -390,7 +411,37 @@ pub(crate) mod tests {
 
     #[test]
     fn from_hex_should_deserialize_a_hex_encoded_string_to_a_dbc() -> Result<(), Error> {
-        let dbc = Dbc::from_hex(DBC_WITH_1_530_000_000)?;
```

Comment:
Is this replacement really necessary?
Cant the public key just be gotten from this preprocessed dbc's dbc content?

Reply:
Not necessary. But this will save us 15 minutes every time we change the DBC struct again.